integration-tests/oauth2-spring/src/test/resources/testRunner.yml

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2021-Present Okta, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+implementation: com.okta.spring.tests.oauth2.spring.SpringApplicationUnderTest
```

Comment:
The OIDC flows are missing: https://github.com/okta/okta-spring-boot/blob/master/integration-tests/oauth2/src/test/resources/testRunner.yml

Reply:
done